azure-bundle/NOTICE

```diff
@@ -7,9 +7,9 @@ The Apache Software Foundation (http://www.apache.org/).
 
 --------------------------------------------------------------------------------
 
-NOTICE for Group: com.fasterxml.jackson.core  Name: jackson-core  Version: 2.13.5
-NOTICE for Group: com.fasterxml.jackson.core  Name: jackson-databind  Version: 2.13.5
-NOTICE for Group: com.fasterxml.jackson.dataformat  Name: jackson-dataformat-xml  Version: 2.13.5
+NOTICE for Group: com.fasterxml.jackson.core  Name: jackson-core  Version: 2.17.2
+NOTICE for Group: com.fasterxml.jackson.core  Name: jackson-databind  Version: 2.17.2
+NOTICE for Group: com.fasterxml.jackson.dataformat  Name: jackson-dataformat-xml  Version: 2.17.2
```

Comment:
Should jackson-dataformat-xml even be mentioned here? We removed it from the license (which I think is correct) based on inspecting the jar.

Reply:
Yes, good point. Let me cleanup here.

Reply:
Done !